Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -958,6 +1127,58 @@ def route_edl() -> Response:
     return resp
 
 
+@APP.route('/log', methods=['GET'])
+def route_edl_log() -> Response:
+    """Show the edl indicators log on '/log' API request. """
+    params = demisto.params()
+
+    cache_refresh_rate: str = params.get('cache_refresh_rate')
+    auth_resp = authenticate_app(params, request.headers)
+    if auth_resp:
+        return auth_resp
+
+    created = datetime.now(timezone.utc)
+    request_args = get_request_args(request.args, params)
+    demisto.debug("Getting log file to show")
+    edl_data_log = ''
+    if os.path.exists(EDL_FULL_LOG_PATH):
+        demisto.debug("found log file")
+        with open(EDL_FULL_LOG_PATH, 'r') as log_file:
+            log_file.seek(0)
+            edl_data_log = log_file.read()
+            log_file.seek(0)
+
+    if len(edl_data_log) == 0:
+        edl_data_log = '# Empty'
+
+    elif request_args.out_format == FORMAT_TEXT:
+        append_str = params.get("append_string")
+        prepend_str = params.get("prepend_string")
+        if append_str:
+            append_str = append_str.replace("\\n", "\n")
+            edl_data_log = f"{edl_data_log}{append_str}"
+        if prepend_str:
+            prepend_str = prepend_str.replace("\\n", "\n")
+            edl_data_log = f"{prepend_str}\n{edl_data_log}"
```

Comment:
Extract to a dedicated function (aooend_prpend_data) or something that you see fit.